src/js/jsx/search/SearchBar.jsx

```diff
@@ -88,50 +91,188 @@ define(function (require, exports, module) {
                     flux.actions.documents.selectDocument(selectedDoc);
                 }
                 break;
+            case "recent-doc":
+                var fileName = idArray[1];
+                
+                if (idArray.length > 2) {
+                    // There are underscores in the file name, so calling split separated the file name
+                    fileName = id.substring(id.indexOf("_") + 1);
+                }
+
+                flux.actions.documents.open(fileName);
+                break;
             }
             this.props.dismissDialog();
         },
 
+        /**
+         * Get the class name for the layer face icon for the layer
+         *
+         * @private
+         * @param {Layer} layer
+         * @return {string}
+         */
+        _getSVGInfo: function (layer) {
+            var iconID = "layer-";
```

Comment:
Yes, moving this to a utility function would probably be a good idea. I messed with this more in the branch `allison/search_filters` though when putting the svgs next to the text box, so it would probably be easier to change it there.